### PR TITLE
Add benchmark of refuel.

### PR DESCRIPTION
**Pull request by giiita**
## Added refuel-json benchmark.
https://github.com/giiita/refuel

### Testing

スクリーンショット 2020-03-20 1 03 51

### CODE REVIEW

**Comment by giiita** [comment]
@yanns Please check it.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Compares performance of several JSON parsers used in Scala:
 - [spray-json](https://github.com/spray/spray-json)
 - [jsoniter-scala](https://github.com/plokhotnyuk/jsoniter-scala)
 - [uJson](http://www.lihaoyi.com/upickle/#uJson)
+- [refuel-josn](https://github.com/giiita/refuel)
```
Comment (plokhotnyuk) [inline README.md]: s/refuel-josn/refuel-json

**Comment by giiita** [comment]
Specific benchmark values ​​may depend on development environment.
Please re-measure if necessary.

File: build.sbt [hunk 2]
```diff
@@ -11,19 +11,20 @@ val circeVersion = "0.9.3"
...
+    "org.json4s"                            %% "json4s-native"        % json4sVersion ::
+    "org.json4s"                            %% "json4s-jackson"       % json4sVersion ::
+    "io.sphere"                             %% "sphere-json"          % "0.9.13"      ::
+    "com.typesafe.play"                     %% "play-json"            % playVersion   ::
+    "io.spray"                              %% "spray-json"           % "1.3.4"       ::
+    "io.argonaut"                           %% "argonaut"             % "6.2.2"       ::
+    "com.github.plokhotnyuk.jsoniter-scala" %% "macros"               % "0.28.0"      ::
+    "com.lihaoyi"                           %% "upickle"              % "0.6.6"       ::
+    "com.phylage"                           %% "refuel-json"          % "1.0.1"       ::
+    Nil
```
Comment (yanns) [inline build.sbt]:
```suggestion
  "io.circe"                              %% "circe-core"           % circeVersion  ::
  "io.circe"                              %% "circe-generic"        % circeVersion  ::
  "io.circe"                              %% "circe-parser"         % circeVersion  ::
  "com.fasterxml.jackson.module"          %% "jackson-module-scala" % "2.9.5"       ::
  "org.json4s"                            %% "json4s-native"        % json4sVersion ::
  "org.json4s"                            %% "json4s-jackson"       % json4sVersion ::
  "io.sphere"                             %% "sphere-json"          % "0.9.13"      ::
  "com.typesafe.play"                     %% "play-json"            % playVersion   ::
  "io.spray"                              %% "spray-json"           % "1.3.4"       ::
  "io.argonaut"                           %% "argonaut"             % "6.2.2"       ::
  "com.github.plokhotnyuk.jsoniter-scala" %% "macros"               % "0.28.0"      ::
  "com.lihaoyi"                           %% "upickle"              % "0.6.6"       ::
  "com.phylage"                           %% "refuel-json"          % "1.0.1"       ::
  Nil
```

File: build.sbt [hunk 3]
```diff
@@ -10,7 +10,7 @@ val circeVersion = "0.9.3"
 
 resolvers += Resolver.bintrayRepo("commercetools", "maven")
 
-libraryDependencies ++=
+libraryDependencies ++=
```
Comment (yanns) [inline build.sbt]:
```suggestion
libraryDependencies ++=
```

Comment (giiita) [reply build.sbt]: I'm sorry, I've responded 🤕

**Pull request merged**